packages/plugin-link-card/README-en.md

```
@@ -0,0 +1,130 @@
+# wangEditor link-card plugin
+
+[中文文档](./README.md)
+
+## Introduction
+
+[wangEditor-next](https://cycleccc.github.io/docs/) link-card plugin.
+
+![](./_img/demo.png)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)